tests/api2/jail.py

```diff
@@ -163,40 +139,42 @@ def test_12_start_jail():
 
 
 def test_13_verify_jail_started():
-    global freeze
-    global freeze_msg
+    global freeze, freeze_msg
     if freeze is True:
         pytest.skip(freeze_msg)
     freeze = False
-    stop_time = time.time() + 600
-    while True:
-        get_job = GET(f'/core/get_jobs/?id={JOB_ID}')
-        assert get_job.status_code == 200, get_job.text
-        job_status = get_job.json()[0]
-        if job_status['state'] in ('RUNNING', 'WAITING'):
-            if stop_time <= time.time():
-                freeze = True
-                freeze_msg = f"Failed to start jail: {JAIL_NAME}"
-                assert False, get_job.text
-                break
-            time.sleep(3)
-        else:
-            results = GET('/jail/')
-            assert results.status_code == 200, results.text
-            assert len(results.json()) > 0, get_job.text
-            assert results.json()[0]['state'].lower() == 'up', results.text
-            break
-
-
-def test_14_export_call():
+    job_status = wait_on_job(JOB_ID, 600)
+    if job_status['state'] in ['TIMEOUT', "FAILED"]:
+        freeze = True
+        freeze_msg = f"Failed to start jail: {JAIL_NAME}"
+    assert job_status['state'] == 'SUCCESS', str(job_status['results'])
+    results = GET(f'/jail/id/{JAIL_NAME}/')
+    assert results.status_code == 200, results.text
+    assert results.json()['state'] == 'up', results.text
+
+
+def test_14_export_jail():
```

Comment:
A jail must not be running when we export it, so that should be ensured here. I think it's running right now ?